static/js/constants.js

```diff
@@ -222,3 +222,16 @@ export const FACET_OPTION_LABEL_KEYS = {
 }
 
 export const FACET_ORDER = [BOOTCAMP_FACET_KEY, STATUS_FACET_KEY]
+
+export const CS_101 = "CS_101"
+export const CS_102 = "CS_102"
+export const CS_150 = "CS_150"
+export const CS_0 = "CS_0"
+
+export const CS_ERROR_MESSAGES = {
+  [CS_101]:
+    "Missing required fields. Please modify your name/address and try again.",
+  [CS_102]: "Invalid data. Please modify your name/address and try again.",
+  [CS_150]: "Please try again.",
```

Comment:
```suggestion
  [CS_150]: "Please try again. If this happens more than once, please contact customer support",
```

Ideally, "contact customer support" would be linked to https://mitbootcamps.zendesk.com/hc/en-us/requests/new -- however, if it takes more than 15m to get HTML to work in the error message, it's not worth iit.

Reply:
@pdpinch I updated the error messages & screenshots